diff --git a/node_modules/@dimforge/rapier3d-compat/rapier_wasm3d.js b/node_modules/@dimforge/rapier3d-compat/rapier_wasm3d.js
index 36fe20b..f863fe3 100644
--- a/node_modules/@dimforge/rapier3d-compat/rapier_wasm3d.js
+++ b/node_modules/@dimforge/rapier3d-compat/rapier_wasm3d.js
@@ -6166,7 +6166,7 @@ async function __wbg_init(module_or_path) {
         if (Object.getPrototypeOf(module_or_path) === Object.prototype) {
             ({module_or_path} = module_or_path)
         } else {
-            console.warn('using deprecated parameters for the initialization function; pass a single object instead')
+            // console.warn('using deprecated parameters for the initialization function; pass a single object instead')
         }
     }
 
